buildSrc/build.gradle

```diff
@@ -35,4 +33,5 @@ dependencies {
     }
     implementation "com.synopsys.integration:blackduck-common:${blackDuckCommonVersion}"
     implementation 'org.springframework:spring-core'
+    implementation gradleApi()
```

Comment:
Why is this line needed? I didn’t need that locally

Reply:
I get the 19 errors without it that are all similar to the following:

```
/Users/danam/WorkGit/Integrations/synopsys-detect/buildSrc/src/main/java/com/synopsys/integration/detect/docs/GenerateDocsTask.java:19: error: package org.gradle.api does not exist
import org.gradle.api.DefaultTask;
                     ^
/Users/danam/WorkGit/Integrations/synopsys-detect/buildSrc/src/main/java/com/synopsys/integration/detect/docs/GenerateDocsTask.java:20: error: package org.gradle.api does not exist
import org.gradle.api.Project;
                     ^

```